### DISCUSSION TITLE
Sonoff T2 Tx - button illumination off

### DISCUSSION THREAD

**Discussion by actionbyte** [arendst/Tasmota#17199] [Q&A]
How can I switch off the LED button lighting on the Sonoff T2 Tx when it is not activated, they are just too bright for me at night? :rolleyes:
When the relay is actuated, the corresponding button should continue to light up.

I don't mean the Wlan/Mqtt status LED, you can configure it with SetOption31, LEDPower or LEDState, but unfortunately I didn't find an option for the buttons themselves.

**Comment by sfromis** [comment]
As the template has no such controls, outside of the status led you've already tried doing things with, it looks like the nightlight led is simply hardwired, meaning that your options are down to modifying the hardware.
